lib/ae_mdw/contract.ex

```diff
@@ -407,7 +407,7 @@ defmodule AeMdw.Contract do
       {:ok, _, _, events} =
         :aec_block_micro_candidate.apply_block_txs_strict(txs_taken, trees_in, env)
 
-      events
+      Enum.reverse(events)
```

Comment:
This needs to be adapted for the reversal:
https://github.com/aeternity/ae_mdw/blob/c19e09b18e2cba71adde92c4963da25ea2e6b81f/lib/ae_mdw/db/sync/contract.ex#L60-L62

Looks like we should have a test case for this dependency on the sequence related to `Chain.create` and `Chain.clone`

Reply:
Thanks for realizing! I've updated events mutations creation and tests accordingly